rs/replicated_state/src/canister_state/system_state/wasm_chunk_store.rs

```diff
@@ -167,7 +167,7 @@ impl WasmChunkStore {
         Ok(hash)
     }
 
-    pub(crate) fn from_checkpoint(data: PageMap, metadata: WasmChunkStoreMetadata) -> Self {
+    pub fn from_checkpoint(data: PageMap, metadata: WasmChunkStoreMetadata) -> Self {
```

Comment:
How did we use to handle it for non-snapshot WasmChunkStore?

Reply:
This function is called from the constructor of `SystemState`, which is in the same crate. Hence the previous `pub(crate)`.